starlark/value.go

```diff
@@ -234,6 +234,14 @@ type Sliceable interface {
 	Slice(start, end, step int) Value
 }
 
+// A MembershipCheckable is a value that supports membership testing
+// via the Has method, which is used by the 'in'/'not in' operator.
+type MembershipCheckable interface {
```

Comment:
Let's call this Container.

// A Container is a value that supports the 'in' operator.

Mention it at L28. (Stupid GitHub PR tool won't let me comment on that line.)


Add `var ( _ Container = (*List)(nil); ...)`  for each known implementation (see L255 for example).